features/apim-analytics-feature/org.wso2.analytics.solutions.apim.analytics.feature/src/main/resources/siddhi-files/apim_health_availability_alert_0.siddhi

```diff
@@ -84,7 +84,7 @@ having AAS.message is null
 insert into ApimAllAlertSummary;
 
 @info(name='Query inserts data to ApimApiHealthMonitorAler')
- from ApiHealthMonitorAlertStreamTemp#window.length(1) as AH left outer join ApimApiHealthMonitorAlert#window.time(10 minutes) as AM
+from ApiHealthMonitorAlertStreamTemp#window.length(1) as AH left outer join ApiHealthMonitorAlertStreamTemp#window.time(10 minutes) as AM
```

Comment:
We can introduce a new in memory stream here - ApimApiHealthMonitorNewTempStream

Reply:
Fixed

Reply:
can we change the join command like this to make it more accurate "from ApiHealthMonitorAlertStreamTemp#window.length(1) as AH left outer join ApiHealthMonitorAlertStream#window.time(10 minutes) as AM"